ui/jstests/test-learning-resource.jsx

```diff
@@ -228,7 +228,15 @@ define(['QUnit', 'jquery', 'reactaddons', 'lodash', 'learning_resources',
 
           var saveButton = $node.find("button")[0];
           React.addons.TestUtils.Simulate.click(saveButton);
+          assert.equal(
```

Comment:
Please only assert a change in state here. We can add a separate test to assert that the state change causes a difference in the rendered output, and so we have two smaller tests rather than one big test.